nibabel/openers.py

```diff
@@ -9,19 +9,66 @@
 """ Context manager openers for various fileobject types
 """
 
-from os.path import splitext
-import gzip
 import bz2
```

Comment:
Have we checked to make sure large `.bz2` files aren't affected by the same upstream bug?